### PR TITLE
PR for Bookstore Demo Theme

### PR DESCRIPTION
**Pull request by aclarkk**
@alimi take a look at this theme, should be ready to pull in. Let me know if you need any changes.

### CODE REVIEW

File: app/views/books/index.html.erb [hunk 1]
```diff
@@ -15,5 +9,5 @@
 </ul>
 
 <% if session[:user_id].present? %>
-  <%= link_to 'Add a book', new_book_path %>
+<%= link_to 'Add a book', new_book_path, class: "button" %>
```
Comment (alimi) [inline app/views/books/index.html.erb]: Can you nest this link back inside the `if`?

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    border-color: $red;
+		&:hover{
+			background: darken($red, 10);
+		}
+	}
+
+  &.grey{
+		background: $greyMed;
+    border-color: $greyMed;
+		&:hover{
```
Comment (alimi) [inline app/assets/stylesheets/application.scss]: There are a few spots throughout the file where the indentation is off. Can you clean it up so the file looks more consistent?

File: app/views/books/show.html.erb [hunk 3]
```diff
@@ -31,7 +31,7 @@
 </ul>
 
 <% if session[:user_id].present? %>
-  <%= link_to 'Edit Book', edit_book_path(@book) %>
-  <%= link_to 'Add a Review', new_book_review_path(@book) %>
-  <%= button_to 'Delete Book', book_path(@book), method: :delete %>
+    <%= link_to 'Edit Book', edit_book_path(@book), class: "button" %>
```
Comment (alimi) [inline app/views/books/show.html.erb]: Can you remove the two extra spaces that were added to the beginning of each line inside the `if`?

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -7,16 +7,31 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
+    <div class="navigation">
```
Comment (alimi) [inline app/views/layouts/application.html.erb:10]: Can you update this file so we're using two spaces per line indent?

**Review by alimi — CHANGES_REQUESTED** [review]
This looks pretty good. I left some feedback to clean up some spacing/indentation to keep the code consistent throughout the app.

Thanks again @aclarkk 🙇🏾

File: app/assets/stylesheets/application.scss (hunk 2 above)
Comment (jgaskins) [reply app/assets/stylesheets/application.scss]: Dem tabs, tho

Comment (clickclickonsal) [reply app/assets/stylesheets/application.scss]: Spaces > Tabs

**Comment by alimi** [comment]
@aclarkk I went ahead and updated the spacing stuff.

b48

**Pull request merged**